mkfile/config/build-tests.mk

```diff
@@ -83,12 +86,12 @@ TEST_INCLUDES_DEBUG   =
 TEST_INCLUDES_RELEASE = 
 
 TEST_INCLUDES_OS = _
-TEST_INCLUDES_OS_WIN32 = 
-TEST_INCLUDES_OS_WIN64 = 
-TEST_INCLUDES_OS_LINUX = 
-TEST_INCLUDES_OS_MACOS = 
-TEST_INCLUDES_OS_OTHER = 
-
+TEST_INCLUDES_OS_WIN32      =
+TEST_INCLUDES_OS_WIN64      =
+TEST_INCLUDES_OS_LINUX      =
+TEST_INCLUDES_OS_MACOS      =
+TEST_INCLUDES_OS_EMSCRIPTEN =
+TEST_INCLUDES_OS_OTHER      =
```

Comment:
I'm fine with undoing the spaces

When doing the same on libavesterra, I started puting the `*_EMSCRIPTEN` after the `*_LINUX`, because they just end up being the same values (at least so far they always have been for me)
Should I still move it to after `*_OTHER*, at the bottom ?

Reply:
@LexouDuck Have you checked out  `git merge -Xignore-space-change` ? I think that would solve such whitespaces-related issues
see https://stackoverflow.com/a/9784089/10263994

Reply:
Okay, will fix this next commit